src/Azure/Orleans.Streaming.AzureStorage/Providers/Streams/AzureQueue/AzureQueueStreamBuilder.cs

```diff
@@ -31,17 +32,12 @@ public SiloAzureQueueStreamConfigurator<TDataAdapter> ConfigureAzureQueue(Action
             this.Configure<AzureQueueOptions>(configureOptions);
             return this;
         }
+
         public SiloAzureQueueStreamConfigurator<TDataAdapter> ConfigureCache(int cacheSize = SimpleQueueCacheOptions.DEFAULT_CACHE_SIZE)
         {
             this.Configure<SimpleQueueCacheOptions>(ob => ob.Configure(options => options.CacheSize = cacheSize));
             return this;
         }
-
-        public SiloAzureQueueStreamConfigurator<TDataAdapter> ConfigurePartitioning(int numOfPartition = HashRingStreamQueueMapperOptions.DEFAULT_NUM_QUEUES)
```

Comment:
Suggest leaving this or adding something like it GenerateQueueNames(..)? to help maintain the original names.  Basically a helper functions that uses AzureQueueUtilities.GenerateQueueNames to build the names for users azure queues.

Reply:
Or maybe add a null check on the names to generate the names if none are provided by the user.

Reply:
I was planning to do the later, if queue names are null , then configure default ones using PostConfigure, just as what I used in my first pass of this PR.

Reply:
I think if we want to retain the original behavior , which is generate a set of queue names by default, we should use PostConfigure, just as how our other default options values are configured. Configure through a GenerateQueueName() method seems clumsy and not really retaining original behavior, since user still need to call it.